versioned/tiered/tiered-impl/src/main/java/com/dremio/nessie/versioned/impl/IdMap.java

```diff
@@ -40,7 +40,7 @@
  * Maintains a map of positions to ids. The map is immutable. Each operation, generates a new map. All maps keep track
  * of their original state so one can see what items changed over time.
  */
-class IdMap implements Iterable<Id> {
+public class IdMap implements Iterable<Id> {
```

Comment:
Why was this made public?

Reply:
I required it at some point but it is no longer needed. I have reverted this change now.